### ISSUE TITLE
get image with the text 

### ISSUE THREAD

**Issue by xxllp** [codelucas/newspaper#255]
hi，I think the nespaper is greet  ,while  I want get the image couple with the  text  in the html ,excepte other image. the code below get all the image ,can it be any tips.
```
from newspaper import Article,images
url = 'http://cnews.chinadaily.com.cn/2016-05/24/content_25439788.htm'

a = Article(url, language='zh') # Chinese

a.download()
a.parse()

print(a.text)
print(a.title)
image= a.images
for img in image:
    print(img)
    print images.fetch_image_dimension(img,None)
```

**Comment by yprez** [comment]
I don't think that what you look for is currently supported.
Maybe you can try comparing against the article_html property?

Something like:
```python
a = Article(url, language='zh', keep_article_html=True)

a.download()
a.parse()

print(a.article_html)  # should contain the <img> tags

for img in a.images:
    if img in a.article_html:
        print(img)
```

Now that I think of it, this functionality is kinda important, maybe it should be builtin.

**Comment by yprez** [comment]
I think this is addressed in #258

Closing this issue for now. Feel free to reopen if you meant something else or still having trouble with it...

**Comment by xxllp** [comment]
I have used the code,but get None.
